### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by dpryan79**
Fix Travis-CI testing and update the installation instructions for anaconda.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,8 +145,8 @@ remember: for support, questions, or feature requests contact: deeptools@googleg
 [bamCoverage]: https://github.com/fidelram/deepTools/wiki/Normalizations#wiki-bamCoverage
 [bamCompare]: https://github.com/fidelram/deepTools/wiki/Normalizations#wiki-bamCompare
 [computeMatrix]: https://github.com/fidelram/deepTools/wiki/Visualizations
-[heatmapper]: https://github.com/fidelram/deepTools/wiki/Visualizations
-[profiler]: https://github.com/fidelram/deepTools/wiki/Visualizations
+[plotHeatmap]: https://github.com/fidelram/deepTools/wiki/Visualizations
+[plotProfile]: https://github.com/fidelram/deepTools/wiki/Visualizations
 [FAQ]: https://github.com/fidelram/deepTools/wiki/FAQ
```
Comment (fidelram) [inline README.md]: We should update this to the new manual location

Comment (dpryan79) [inline README.md]: Indeed, we should now

Comment (vivekbhr) [inline README.md]: by the way, aren't we supposed to get rid of the wiki completely??

Comment (dpryan79) [inline README.md]: No, we'll keep it around forever for <2.0 releases. We need to add a link at the top of each page to read the docs, though (it's on my to do list).

**Pull request merged**